torch/_tensor_docs.py

```diff
@@ -873,6 +887,18 @@ def add_docstr_all(method, docstr):
 See :func:`torch.clone`
 """.format(**common_args))
 
+add_docstr_all('coalesce',
+               r"""
+coalesce() -> Tensor
+
+If :attr:`self` is a sparse COO tensor (i.e., with
```

Comment:
"Returns a coalesced copy of a sparse self. Throws an error if self is not sparse."

Reply:
The suggestion is inaccurate because when `self` is already coalesced then `self` is returned (no copy).

Using:
```
Returns a coalesced copy of :attr:`self` if :attr:`self` is an
:ref:`uncoalesced tensor <sparse-uncoalesed-coo-docs>`.

Returns :attr:`self` if :attr:`self` is a coalesced tensor.

Throws an error if :attr:`self` is not a sparse COO tensor.
```